.github/CODEOWNERS

```diff
@@ -49,7 +49,8 @@ src/lib/core/datetime/**           @mmalerba
 # CDK
 src/cdk/coercion/**                @jelbourn
 src/cdk/rxjs/**                    @jelbourn
-src/cdk/observe-content/**         @crisbeto @jelbourn
+src/cdk/observe-content/**         @jelbourn @crisbeto
+src/cdk/collections/**             @jelbourn @andrewseguin
```

Comment:
For sure, added you as an owner. I removed `@docs-private` as well since we are advising users to apply it for their tables.